bbz-project/src/components/RecordsListComponents/RecordComps/Record.js

```diff
@@ -29,36 +50,50 @@ const Record = ({ data, keysAmount }) => {
   };
 
   return (
-    <animated.div style={heightAnim} className={"recordMainBck w-100"}>
+    <animated.div
+      style={enableAnim ? heightAnim : {}}
+      className={"recordMainBck w-100"}
+    >
       <div
         className={"recordBck w-100 d-flex flex-row position-relative"}
         onClick={handleRecordClick}
       >
-        {keys.map((item, key) =>
-          key <= keysAmount ? (
-            <div
-              key={key}
-              className={"recordFieldBck h-100 d-flex align-items-center "}
-              style={{
-                width:
-                  keysAmount === keys.length - 1
-                    ? FIELD_WIDTHS[key]
-                    : FIELD_WIDTHS_SHORT[key],
-              }}
-            >
-              {data[item]}
-            </div>
-          ) : (
-            <div key={key} style={{ display: "none" }} />
-          )
-        )}
-        <RecordOptions visible={optionsVisibility} />
+        {valuesToDisplay.map((item, index) => (
+          <div
+            key={index}
+            className={
+              "key" +
+              (index + 1) +
+              " recordFieldBck h-100 align-items-center"
```

Comment:
Template literals may be used to make this code simpler and easier to comprehence: e.g. `String text ${expression}`

Reply:
https://stackoverflow.com/questions/3304014/how-to-interpolate-variables-in-strings-in-javascript-without-concatenation